Dockerfile.fedora25

```diff
@@ -0,0 +1,18 @@
+FROM fedora:25
```

Comment:
`fedora:26` now

Reply:
Sorry for the delay. Yes I will change it for the fedora 26. Sorry for the late reply.